docs/website/docs/HardWonWisdom.md

```diff
@@ -112,3 +112,196 @@ REQ2: If L≤k (see REQ1) and L<R, we must validate at least L+1 of their header
 The most demanding case of REQ2 is L=k: at most we'll need to validate k+1 of the peer's headers.
 Thus using HCG window as Stability Window ensures that forecasting can't disrupt REQ2 when the peer is serving honest blocks.
 (We can tick the intersection ledger state to their first header, and then forecast 3k/f from there which by HCG will get us at least the remaining k headers if they're serving an honest chain.)
+
+## Which slots/times/epochs can we translate?
+
+TODO Confirm and emphasize that this exact same logic is used by the node itself to determine which slot the current wall clock is in.
```

Comment:
You mean whether `hardForkBlockchainTime` uses that mechanism? In that case, the answer is positive.